base/applications/mspaint/lang/en-US.rc

```diff
@@ -242,4 +241,6 @@ BEGIN
     IDS_ITALIC "Italic"
     IDS_UNDERLINE "Underline"
     IDS_VERTICAL "Vertical"
+    IDS_LEGALRESUNITS "dot/inch" /* Don't translate this line */
```

Comment:
Looking again at the code, it would appear possible to have that string (that remains untranslated) as the default (resource) string of within the corresponding dialog static text control. That would allow not growing the string table unnecessarily.
Then in the `CAttributesDialog::OnInitDialog` function could do a simple `GetDlgItemText` call on the corresponding static text control, instead of calling `strUnit.LoadString(hProgInstance, IDS_LEGALRESUNITS);`
(Note however that this would work only in the OnInitDialog function -- when the dialog is created and default contents being initialized -- and nowhere else, since the static text would then change.)

Reply:
dot per inch (dpi) better no?

Reply:
OK, fixed.